pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -1232,7 +1232,7 @@ assertNoAdditions {
   };
 
   follow-md-links-nvim = super.follow-md-links-nvim.overrideAttrs {
-    dependencies = [ self.nvim-treesitter ];
+    dependencies = [ self.nvim-treesitter-legacy ];
```

Comment:
I don't see this even requiring `nvim-treesitter` it's logic is based off `vim.treesitter` API.